### DISCUSSION TITLE
Subdivision Modeling and Derivation Modeling

### DISCUSSION THREAD

**Discussion by tautologias345** [godotengine/godot-proposals#6027] [3D]
With Godot 3.5.1, to create mesh, I'm capable to use Primitive Modeling and Solid Modeling. Solid Modeling is Construtive Solid Geometry. I suggest Subdivision Modeling and Derivation Modeling for Godot, because Blender 3.3 already supports.

**Comment by Calinou** [comment]
Godot isn't 3D modelling software – it's a game engine. That said, any add-on is free to implement this feature.

If you're looking to perform subdivision on existing 3D meshes, see https://github.com/godotengine/godot-proposals/issues/784.
